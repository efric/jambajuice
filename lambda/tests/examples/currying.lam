(\x y. plus (plus x y) x) one two

-- Should reduce to plus (plus one two) one
