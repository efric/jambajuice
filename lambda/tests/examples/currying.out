(\x . \y . plus (plus x y) x) one two
